styles/index.css

```diff
@@ -0,0 +1,77 @@
+body {
+  color: hsl(309, 42%, 32%);
```

Comment:
your page was responsive and clean. I like that you also changed the text color with the change of the sky.